libraries/Microsoft.Bot.Schema/Teams/NotificationRecipientFailureInfo.cs

```diff
@@ -6,28 +6,28 @@
 namespace Microsoft.Bot.Schema.Teams
 {
     /// <summary>
-    /// Information regarding failure to notify a recipient of a <see cref="TeamsMeetingNotification"/>.
+    /// Information regarding failure to notify a recipient of a <see cref="TargetedMeetingNotification"/>.
     /// </summary>
-    public class TeamsMeetingNotificationRecipientFailureInfo
+    public class NotificationRecipientFailureInfo
```

Comment:
Do you think prefixing it with would `Meeting` be a good idea? I think so since 'notification' in MS Team's context means the message notification, and not the lesser known meeting notification.

Reply:
This naming aligns with what is defined in SMBA. 
Do we want to align with the name in SMBA or prefer to add `Meeting `prefix for NotificationRecipientFailureInfo?
@YunnyChung, do you have any though on this?

![image](https://user-images.githubusercontent.com/7191986/214738441-857aedf0-c0fd-4169-84d8-c2f00ce97312.png)

Reply:
We should make a decision from the perspective of developers, not what is internally defined

Reply:
Seems we all agreed on updating the name to `MeetingNotificationRecipientFailureInfo`.
Will update it in the next iteration.